evm/vm/forks/sharding/vmc_handler.py

```diff
@@ -15,11 +15,14 @@
 )
 
 from evm.vm.forks.sharding.config import (
-    DEPOSIT_SIZE,
-    GASPRICE,
-    TX_GAS,
+    get_sharding_config,
 )
 
+sharding_config = get_sharding_config()
+DEPOSIT_SIZE = sharding_config['DEPOSIT_SIZE']
+GAS_PRICE = sharding_config['GAS_PRICE']
+DEFAULT_GAS = sharding_config['DEFAULT_GAS']
```

Comment:
Rather than storing these at the module level, I would recommend storing the `sharding_config` as a property of the `VMC` instance.  Populate it in the `__init__` method and then pull values from it from whatever property you set it to:

```python
def sample(self, shard_id, gas=None):
    if gas is None:
        gas = self.config['DEFAULT_GAS']
    ...
```

Reason for this pattern is:

* Allow for two independent VMC contract instances which use different config values.
* Allow for environment configuration to be set at runtime *after* module loading has occurred.

Reply:
That's a great idea! I got you're point. In my code it was hard for two contract to use different config, since `sharding_config` is set when the module is executed, not when the VMC instance is created. 
Will modify it. Thank you:)